### DISCUSSION TITLE
Runtime detectBrowserLanguage.cookieDomain modification

### DISCUSSION THREAD

**Discussion by MissedWarrior** [nuxt-modules/i18n#1853] [Q&A]
Hey guys!

Is it possible to modify detectBrowserLanguage.cookieDomain in runtime?

Current problem: we have multiple second-level domains in our organization and we need same build for each of them. Also, we have multiple different apps that are working on different third-level domains and we want to sync selected language via cookies. So, my idea is somehow modify cookieDomain in runtime to match it to second-level domain of the current host. Is it possible to achieve?

Thank you :)

P.S. Ssr option set to false, app is working only on client

**Comment by felixgabler** [comment]
Hey, did you ever find a solution to this? We are encountering the same issue
